web/src/search_suggestion.ts

```diff
@@ -729,42 +735,74 @@ function get_operator_suggestions(last: NarrowTerm): Suggestion[] {
     });
 }
 
+// One full search suggestion can include multiple search terms, based on what's
+// in the search bar.
+type SuggestionLine = Suggestion[];
```

Comment:
Would it be good to rename `Suggestion` to something like `SuggestionTerm` to more clearly differentiate from `SuggestionLine`?

Reply:
Possibly; might be best to defer until we've completed the pill transition just to void unnecessary churn.